app/models/concerns/vacancy_candidate_specification_validations.rb

```diff
@@ -12,6 +12,10 @@ module VacancyCandidateSpecificationValidations
                                if: proc { |model| model.qualifications.present? }
   end
 
+  def upload_feature_enabled?
+    UploadDocumentsFeature.enabled?
+  end
+
```

Comment:
Curious as to why you introduced a method instead of just using `UploadDocumentsFeature.enabled?` directly where needed?

Reply:
Apologies, this is out of scope for the PR - rebased and re-pushed to reflect this.